pkg/cmd/server/origin/controller.go

```diff
@@ -143,6 +148,14 @@ func (c *MasterConfig) NewOpenShiftControllerPreStartInitializers() (map[string]
 	// this matches the upstream name
 	ret["serviceaccount-token"] = saToken.RunController
 
+	hpaController := kubernetes.HorizontalPodAutoscalerControllerConfig{
```

Comment:
Find a way to do this inside of michal's method that's building the entire set of kube init functions.